src/graph/helpers/create-default-properties.ts

```diff
@@ -0,0 +1,47 @@
+import { COVER_PROPERTY, DESCRIPTION_PROPERTY, NAME_PROPERTY } from '~/src/core/ids/system.js';
+import { Relation } from '../../relation.js';
+import { Triple } from '../../triple.js';
+import type { DefaultProperties, Op } from '../../types.js';
+
+export const createDefaultProperties = (params: DefaultProperties & { entityId: string }) => {
+  const { entityId, name, description, cover } = params;
+  const ops: Op[] = [];
+
+  if (name) {
+    // set property "Name" to the provided name
+    const nameTripleOp = Triple.make({
+      entityId,
+      attributeId: NAME_PROPERTY,
+      value: {
+        type: 'TEXT',
+        value: name,
+      },
+    });
+    ops.push(nameTripleOp);
+  }
+
+  if (description) {
+    // set property "Description" to the provided description
+    const descriptionTripleOp = Triple.make({
+      entityId,
+      attributeId: DESCRIPTION_PROPERTY,
+      value: {
+        type: 'TEXT',
+        value: description,
+      },
+    });
+    ops.push(descriptionTripleOp);
+  }
+
+  if (cover) {
+    // add ID of cover "Image" to property "Cover"
+    const relationOp = Relation.make({
+      fromId: entityId,
+      relationTypeId: COVER_PROPERTY,
+      toId: cover,
```

Comment:
The cover is actually an entity id and not just a string. Not sure the best API for users creating images yet. We do have an `Image.make(...)` function but it requires that they've uploaded their image to IPFS already and have a URL for it.

Reply:
As mentioned here https://github.com/graphprotocol/grc-20-ts/pull/31#discussion_r1966066174 all IDs at the moment are strings. Is this fine or what should I change?

Reply:
Responded in this comment thread https://github.com/graphprotocol/grc-20-ts/pull/31#discussion_r1966101517